### ISSUE TITLE
Dokku vs Heroku - hosting pros and cons

### ISSUE THREAD

**Issue by dpritchett** [memtech/elvis#53]
cc @geekblogtv @joshwlewis @vongrippen 

James offers to put Elvis up on a droplet running dokku.  Josh is comping elvis already but performance has been a little unstable lately.

Lots of the instability is really just untested monkey patching in Elvis's codebase and not Heroku issues.

Thoughts?

**Comment by vongrippen** [comment]
I mean... So long as we can continue with the autodeploys and easy env management.

I need to set aside time to work on elvis and get him cleaned up, the patches are really breaking him.

**Comment by joshwlewis** [comment]
Yeah, the fact that we all have access to config, deploy, and restart Elvis is really nice. As long as we don't lose that, I'm fine with it.
